src/main.cpp

```diff
@@ -60,4 +60,4 @@ int main(int argc, char** argv)
 	game.run();
 
 	return 0;
-}
\ No newline at end of file
+}
```

Comment:
You shouldn't `git add` it before you commit, and then it wouldn't be included. There's nothing to do about it now, but don't worry, this is how it should be. :)